packages/core/admin/admin/src/pages/Settings/components/SettingsNav.tsx

```diff
@@ -24,6 +67,12 @@ const CustomIcon = styled(Lightning)`
   }
 `;
 
+const SubNavStyled = styled(SubNav)`
+  overflow-y: auto !important;
```

Comment:
please try to avoid the use of !important, it is recommended to try to avoid it even in the mdn documentation https://developer.mozilla.org/en-US/docs/Web/CSS/important#best_practices